src/libsync/creds/oauth.cpp

```diff
@@ -101,7 +106,6 @@ void OAuth::startAuthentication()
                     { QStringLiteral("code"), args.queryItemValue(QStringLiteral("code")) },
                     { QStringLiteral("redirect_uri"), QStringLiteral("http://localhost:%1").arg(_server.serverPort()) },
                     { QStringLiteral("code_verifier"), _pkceCodeVerifier },
-                    { QStringLiteral("scope"), args.queryItemValue(QStringLiteral("scope")) },
```

Comment:
Wait, was that not the correct thing to do?

Reply:
I have no idea and it looks like we would need the full set anyhow....
@DeepDiver1975 ?

Reply:
But then we would need to use `SCOPE()` here.
If you don't use it here, there is also no reason to use in the other call.

Reply:
It's now set in postTokenRefresh